In file included from mibgroup/iwlib.h:139,
                 from mibgroup/ieee802dot11.c:29:
.../include/linux/if.h:140:8: error: redefinition of 'struct ifmap'
 struct ifmap {
        ^~~~~
In file included from mibgroup/iwlib.h:33,
                 from mibgroup/ieee802dot11.c:29:
.../include/net/if.h:111:8: note: originally defined here
 struct ifmap
        ^~~~~
In file included from mibgroup/iwlib.h:139,
                 from mibgroup/ieee802dot11.c:29:
.../include/linux/if.h:174:8: error: redefinition of 'struct ifreq'
 struct ifreq {
        ^~~~~
In file included from mibgroup/iwlib.h:33,
                 from mibgroup/ieee802dot11.c:29:
.../include/net/if.h:126:8: note: originally defined here
 struct ifreq
        ^~~~~
In file included from mibgroup/iwlib.h:139,
                 from mibgroup/ieee802dot11.c:29:
.../include/linux/if.h:223:8: error: redefinition of 'struct ifconf'
 struct ifconf  {
        ^~~~~~
In file included from mibgroup/iwlib.h:33,
                 from mibgroup/ieee802dot11.c:29:
.../include/net/if.h:176:8: note: originally defined here
 struct ifconf
        ^~~~~~

--

--- a/agent/mibgroup/iwlib.h
+++ b/agent/mibgroup/iwlib.h
@@ -29,9 +29,9 @@
 #include <unistd.h>
 #include <netdb.h>		/* gethostbyname, getnetbyname */
 #include <net/ethernet.h>	/* struct ether_addr */
-#ifdef HAVE_NET_IF_H
-#include <net/if.h>
-#endif
+//#ifdef HAVE_NET_IF_H
+//#include <net/if.h>
+//#endif
 #include <sys/time.h>		/* struct timeval */
 #include <unistd.h>
 

